src/asteroids.ts

```diff
@@ -209,14 +222,14 @@ function moveAsteroids() {
     roids[i].y += roids[i].yv;
     // handle edge of screen
     // if (roids[i].x < 0 - roids[i].r) {
-    //     roids[i].x = cvs.width + roids[i].r;
-    // } else if (roids[i].x > cvs.width + roids[i].r) {
+    //     roids[i].x = CVS.width + roids[i].r;
+    // } else if (roids[i].x > CVS.width + roids[i].r) {
     //     roids[i].x = 0 + roids[i].r;
     // }
 
     // if (roids[i].y < 0 - roids[i].r) {
-    //     roids[i].y = cvs.height + roids[i].r;
-    // } else if (roids[i].y > cvs.height + roids[i].r) {
+    //     roids[i].y = CVS.height + roids[i].r;
+    // } else if (roids[i].y > CVS.height + roids[i].r) {
```

Comment:
it's better not to save dead code in projects

we have git for that

Reply:
Agreed!

@vivagable2 I think we want to remove all the handle edge of screen code because we are no longer having asteroids or ships pop in and out of frame.